rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -128,7 +128,7 @@
 
     private static LRAData convert(LRAStatusHolder lra) {
         return new LRAData(lra.getLraId(), lra.getClientId(),
-                lra.getStatus().name(),
+                lra.getStatus().name(), lra.getLraStatusReason(),
```

Comment:
rename `getLraStatusReason` to `getStatusReason` since the field is already part of an LRA data structure. Similarly for the setter.